client/templates/adminMenu/adminMenu.html

```diff
@@ -1,4 +1,4 @@
-<div layout="row">
+<div layout="row" layout-align="center center">
```

Comment:
There's something weird going on with this template, two divs are opened but three are closed.